<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterlisImport</class>
 <widget class="QDialog" name="InterlisImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Project</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_10">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>88</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Source</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="type_combo_box">
     <item>
      <property name="text">
       <string>Interlis (PostGIS)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Interlis (GeoPackage)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PostGIS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GeoPackage</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="6" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QProgressBar" name="progress_bar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QTextBrowser" name="txtStdout">
         <property name="toolTip">
          <string/>
         </property>
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="pg_config">
         <property name="title">
          <string>PostgreSQL</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Schema</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="pg_host_line_edit">
            <property name="text">
             <string>localhost</string>
            </property>
            <property name="placeholderText">
             <string>Database Hostname</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>User</string>
            </property>
           </widget>
          </item>
          <item row="9" column="2">
           <widget class="QLineEdit" name="pg_password_line_edit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>[Leave empty to use system password]</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Host</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="pg_schema_line_edit">
            <property name="placeholderText">
             <string>[Leave empty to load all schemas in the database]</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Database</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="pg_port_line_edit">
            <property name="placeholderText">
             <string>[Leave empty to use standard port 5432]</string>
            </property>
           </widget>
          </item>
          <item row="10" column="2">
           <widget class="QCheckBox" name="pg_use_super_login">
            <property name="text">
             <string>Use logins from settings (superuser) for schema generation</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="pg_database_line_edit">
            <property name="placeholderText">
             <string>Database Name</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLineEdit" name="pg_user_line_edit">
            <property name="placeholderText">
             <string>Database Username</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="gpkg_config">
         <property name="title">
          <string>GeoPackage</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Database File</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="gpkg_file_line_edit"/>
          </item>
          <item row="0" column="3">
           <widget class="QToolButton" name="gpkg_file_browse_button">
            <property name="toolTip">
             <string>Browse GeoPackage files</string>
            </property>
            <property name="text">
             <string>…</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="ili_config">
         <property name="title">
          <string>Interlis</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="5" column="2" colspan="2">
           <widget class="QPushButton" name="ili2db_options_button">
            <property name="text">
             <string>Advanced Options</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="3">
           <widget class="QgsProjectionSelectionWidget" name="crsSelector">
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="crs_label">
            <property name="toolTip">
             <string>Coordinate Reference System</string>
            </property>
            <property name="text">
             <string>CRS</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <widget class="QToolButton" name="ili_file_browse_button">
            <property name="toolTip">
             <string>Browse Interlis files</string>
            </property>
            <property name="text">
             <string>…</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Interlis File</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="CompletionLineEdit" name="ili_models_line_edit" native="true"/>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="ili_file_line_edit">
            <property name="placeholderText">
             <string>[Optional]</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QToolButton" name="multiple_models_button">
            <property name="toolTip">
             <string>Select multiple Interlis models</string>
            </property>
            <property name="text">
             <string>…</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Models</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="4">
           <widget class="QLabel" name="toml_file_info_label">
            <property name="maximumSize">
             <size>
              <width>580</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="mssql_config">
         <property name="title">
          <string>Mssql</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Host</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="mssql_host_line_edit">
            <property name="text">
             <string>localhost</string>
            </property>
            <property name="placeholderText">
             <string>Database Hostname</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Instance</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="mssql_instance_line_edit">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>[Leave empty to use default instance]</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="mssql_port_line_edit">
            <property name="placeholderText">
             <string>[Leave empty to use dinamic or standard port 1433]</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Database</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="mssql_database_line_edit">
            <property name="placeholderText">
             <string>Database Name</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Schema</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="mssql_schema_line_edit">
            <property name="placeholderText">
             <string>[Leave empty to load all schemas in the database]</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>User</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="mssql_user_line_edit">
            <property name="placeholderText">
             <string>Database Username</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLineEdit" name="mssql_password_line_edit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>[Leave empty to use system password]</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CompletionLineEdit</class>
   <extends></extends>
   <header>QgisModelBaker.gui.options</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>type_combo_box</tabstop>
  <tabstop>ili_file_line_edit</tabstop>
  <tabstop>ili_file_browse_button</tabstop>
  <tabstop>ili_models_line_edit</tabstop>
  <tabstop>multiple_models_button</tabstop>
  <tabstop>crsSelector</tabstop>
  <tabstop>ili2db_options_button</tabstop>
  <tabstop>pg_host_line_edit</tabstop>
  <tabstop>pg_port_line_edit</tabstop>
  <tabstop>pg_database_line_edit</tabstop>
  <tabstop>pg_schema_line_edit</tabstop>
  <tabstop>pg_user_line_edit</tabstop>
  <tabstop>pg_password_line_edit</tabstop>
  <tabstop>gpkg_file_line_edit</tabstop>
  <tabstop>gpkg_file_browse_button</tabstop>
  <tabstop>txtStdout</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InterlisImport</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InterlisImport</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
